src/cli.erl

```diff
@@ -0,0 +1,69 @@
+-module(cli).
+-export([main/1]).
+-include("gradualizer.hrl").
+
+-spec main([string()]) -> ok | no_return().

```

Comment:
`ok | no_return() = ok`

Reply:
Ah, I'll move the non-termination condition to a normal comment instead.